src/components/Footer.tsx

```diff
@@ -54,7 +54,10 @@ export default function Footer() {
             </a>
           </div>
         </div>
-
+        <div className="flex flex-col  hidden md:block md:px-[25px] lg:px-[80px]">
+          <p className="text-center text-white md:text-xs lg:text-base font-thin mb-5">{footer.sponsorText}</p>
+          <img className="" src="/img/omegapoint_logo.svg"></img>
+        </div>
```

Comment:
Two things: (see image below for more)
1. The order of the text and image should be inverted.
2. The image seems a tad bit bigger than it should be to me.

Reference image:
![image](https://github.com/datasektionen/ddagen/assets/66337956/3c1c67ae-7b74-4917-81d0-97f514f67078)

How it looks on my screen:
![image](https://github.com/datasektionen/ddagen/assets/66337956/55181858-43cb-44cf-95a2-4da2c3877a22)

Reply:
@ernum I made the logo smaller now.
But the text should be above the image, I think.
![bild](https://github.com/datasektionen/ddagen/assets/58601215/6ba0fbbe-5cde-44e2-8b95-199d9f03e0b0)